puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/foremanCentosJob.groovy

```diff
@@ -39,6 +39,7 @@ pipeline {
                 runPlaybook(
                     playbook: 'foreman-infra/ci/centos.org/ansible/fetch_debug_files.yml',
                     inventory: cico_inventory('./'),
+                    extraVars: ["workspace=/home/foreman/workspace/${env.JOB_NAME}/debug"],
```

Comment:
I thought the workspace was already an env var. Can't you reuse that? I also think pointing workspace to a subdir of the actual workspace could be a bit misleading.

Reply:
Good call, updated